index.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Assignment 6 </title>
+    <meta charset="utf-8">
+   
+    
+</head>
+
+<body>
+<!-- Scott Fincham
+assignment06
+INFO2340.WW
+Thoendel
+Winter 2019
+-->
+<h1> Hello, world!  </h1>
+  <div>   Allow me to introduce myself! My name is <?php echo "Scott Fincham!" ;?></div>

```

Comment:
Nice php statement! Complete with those needed quotation marks and a semicolon! 👍